ui/lib/kv/addon/components/kv-list-filter.hbs

```diff
@@ -19,7 +19,7 @@
   </div>
 </div>
 {{#if this.filterIsFocused}}
-  {{#if this.filterMatchesASecretPath}}
+  {{#if this.isFilterMatch}}
```

Comment:
could these be combined?
```suggestion
  {{#if (and this.filterIsFocused this.isFilterMatch)}}
```

Reply:
yes they could! Good catch. I was going step by step on the logic and missed this clean up.